app/src/test/java/com/gholem/moneylab/features/editTransaction/viewmodel/EditTransactionViewModelTest.kt

```diff
@@ -68,24 +77,26 @@ class EditTransactionViewModelTest {
             expectNoEvents()
         }
     }
-    ///???? HOW TO MOCK LATEINIT VAR
+
     @Test
     fun `verify invocations on saveEditedTransaction method call`() = runTest {
```

Comment:
Didn't checked what exactly is missing there, but a lot of methods of view model is not covered - add tests for them please (and check also all other view model tests)